lib/ControlledVocab/ControlledVocab.js

```diff
@@ -456,10 +456,12 @@ class ControlledVocab extends React.Component {
               onCreate={this.actuators.onCreate}
               onDelete={this.showConfirmDialog}
               isEmptyMessage={
-                <FormattedMessage
-                  id="stripes-smart-components.cv.noExistingTerms"
-                  values={{ terms: this.props.label }}
-                />
+                '?condition?'
```

Comment:
Maybe 
```
const values = props.resources.values;
const condition = values.hasLoaded && values.records?.length === 0;
```

Reply:
Yeah, exactly! I could not guess to check every property of the `values`! Thanks, @zburke, for the hint.
The only thing is the `hasLoaded` property is probably not suitable for us, because it remains `false` if no data has been received. But `isPending` is great for us.